src/main/java/com/metamx/metrics/JvmMonitor.java

```diff
@@ -156,10 +53,23 @@ public JvmMonitor(Map<String, String[]> dimensions)
   @Override
   public boolean doMonitor(ServiceEmitter emitter)
   {
+    // jvm/mem
+    emitJvmMemMetrics(emitter);
+
+    // direct memory usage
+    emitDirectMemMetrics(emitter);
+
+    // jvm/gc
+    emitGcMetrics(emitter);
+
+    return true;
+  }
+
+  @Deprecated
```

Comment:
Maybe makes sense to leave a comment right in the code, that these metrics are superseded but other metrics, emitted below.